pandas/io/stata.py

```diff
@@ -1831,6 +1831,11 @@ class StataWriter(StataParser):
     dataset_label : str
         A label for the data set.  Should be 80 characters or smaller.
 
+    .. versionadded:: 0.19.0
+    variable_labels : dict
```

Comment:
not necessarily in this case, but it should be after the keyword, and indented to satisfy sphinx.

I will fix it in a general doc fixups PR shortly